packages/sitecore-jss-nextjs/src/edge/redirects-middleware.ts

```diff
@@ -79,13 +79,15 @@ export class RedirectsMiddleware {
    * @returns Promise<RedirectInfo | undefined>
    * @private
    */
-  private async getExistsRedirect(req: NextRequest): Promise<RedirectInfo | undefined> {
+  private async getExistsRedirect(req: NextRequest): Promise<RedirectInfo | void> {
     const redirects = await this.redirectsService.fetchRedirects();
```

Comment:
`fetchRedirects` returns array by the return type. We shouldn't expect something except of array or `fetchRedirects` should be changed

Reply:
fixed